src/event.h

```diff
@@ -212,6 +213,15 @@ struct event_spec<event_type::awakes_dark_wyrms> : event_spec_empty {};
 template<>
 struct event_spec<event_type::becomes_wanted> : event_spec_character {};
 
+template<>
+struct event_spec<event_type::broken_bone> {
+    static constexpr std::array<std::pair<const char *, cata_variant_type>, 2> fields = {{
+            { "character", cata_variant_type::character_id },
+            { "part", cata_variant_type::body_part },
+        }
+    };
+};
+
```

Comment:
This can be a future thing, but just to give you ideas:
If you wanted to implement the "have all limbs broken simultaneously" achievement suggestion, that is possible.  You'd do it by adding a `bool` member here which is `true` for broken and `false` for healed, add suitable event triggers in the code for when limbs heal and send the event with `false`.  Then you can take the event stream for a particular limb, write a statistic that's observing the last value in that stream, and it will tell you whether the bone is currently broken.

Reply:
Thanks for the tip, I'll check it out.